tests/factories.py

```diff
@@ -91,10 +91,8 @@ def uri(stub):
         return "http://example.com/document_{n}".format(n=stub.random_number)
 
     @factory.LazyAttribute
-    def target(stub):
+    def target_selectors(stub):
         return [{
```

Comment:
Ditto here. This is now returning an object which doesn't look like anything that will ever be returned from `target_selectors` on a real `Annotation` object.